src/main/js/portal/main/models/CartItem.ts

```diff
@@ -167,14 +168,15 @@ export default class CartItem {
 		return this._keyValPairs[key];
 	}
 
-	getNewUrl(keyVal: IdxSig){
+	getNewUrl(keyVal: IdxSig | PreviewSettings){
```

Comment:
This seems more applicable, I think?
```suggestion
	getNewUrl(keyVal: Partial<PreviewSettings>){
```